include/array.h

```diff
@@ -1,5 +1,8 @@
 #pragma once
 
+#include <iostream>
```

Comment:
so we need that here?

Reply:
That's there because of the `<<` method defined in that file.  It should have always been there (things broke with some header cleanup).

Reply:
Should have caught this.